Formula/n98-magerun.rb

```diff
@@ -0,0 +1,17 @@
+class N98Magerun < Formula
+  desc "Swiss army knife for Magento developers, sysadmins and devops."
+  homepage "http://magerun.net/"
+  url "https://files.magerun.net/n98-magerun-1.97.22.phar"
+  version "1.97.22"
```

Comment:
You can remove this line as the version will be auto-detected from the URL. You can check for issues like these locally by running `brew audit --strict --online n98-magerun`.